merlin/models/tf/blocks/mlp.py

```diff
@@ -86,15 +86,19 @@ def MLPBlock(
         The name of the block.
     """
 
+    if isinstance(activation, list) and len(activation) != len(dimensions):
+        raise ValueError("Activation list should be of same length as Dimensions")
```

Comment:
Could be useful to add the lenghts of both lists to the error-message.

Reply:
Yes, correct. I have added that information.